hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/spi/state/StateKey.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.node.app.spi.state;
+
+/** IDs for representing different state's keys. */
+public enum StateKey {
+    ACCOUNT_STORE,
+    ALIASES_STORE,
+    TOKEN_STORE
+}
```

Comment:
These must not be here. Nothing in `com.hedera.node.api.spi` should have any notion of the concrete service types. What stores exist as part of the CryptoService are no business of the SPI, or any other module. Only the crypto module should have any concept of what kinds of stores it has.

Reply:
Understood. Will move this to `com.hedera.node.app.spi.state.impl` ?

Reply:
Removed `StateKey` and used Key as `String` for initial implementations